app/[lang]/(inferenceai)/inferenceai/data.ts

```diff
@@ -105,6 +106,32 @@ export const navInferenceai = (lang: SupportedLanguage) => {
   return [
     { label: inferenceaiNavItems0Label(lang), href: "#how-it-works" },
     { label: inferenceaiNavItems1Label(lang), href: "#what-you-get" },
+    {
+      label: inferenceaiNavItems5Label(lang),
+      href: "#",
+      children: [
```

Comment:
Please use [this data](https://github.com/hyperjumptech/hyperjump.tech/blob/main/app/%5Blang%5D/(inferenceai)/inferenceai/data.ts#L209) to get all the solutions.